bgpd/bgp_route.h

```diff
@@ -278,6 +278,7 @@ struct bgp_path_info {
 
 	/* Uptime.  */
 	time_t uptime;
+	time_t rib_uptime;
 
```

Comment:
I am worried by the additional memory cost for a value that is used only for BMP Loc.
Could this value be moved to path_info_extra ?

Reply:
you're right, this needs to be changed
at first I didn't want to include the RIB uptime commits in my PR as they're still WIP but I forgot to exclude them 

would it be okay with you to merge the code as is and to open an issue to resolve this after ?

Reply:
moved rib_uptime to bgp_path_info_extra and written in header in https://github.com/FRRouting/frr/commit/aa778179ff888500b345ba1f14e95046d5c2c529